### ISSUE TITLE
bit perfect audio?

### ISSUE THREAD

**Issue by rjkroege** [iina/iina#2947]
I'm wondering if there is a way to have IINA automatic set the CoreAudio bit rate and format to send the PCM stream through unmodified to an external DAC? (Seems to be sometimes called "bit-perfect" on the Internet)

Maybe mpv needs something like https://github.com/mpv-player/mpv/pull/6791 first? Or maybe something like this already works and I'm just being noob-y? Or I just have to add an mpv flags?

Anyway: I tested this with a directory of assorted FLACs, an internal DAC and IINA doesn't adjust the output bitrate (according to the DAC)

**Comment by macdavis** [comment]
> Or I just have to add an mpv flags?

Yes. Use either `ao=coreaudio_exclusive`, or `coreaudio-change-physical-format=yes`.
